build.gradle

```diff
@@ -343,17 +381,34 @@ for (publishedProject in publishedProjects) {
             maven(MavenPublication) {
                 from project.components.java
                 pom {
+                    groupId = project.group
+                    artifactId = project.name
+                    version = project.version
+                    packaging = "jar"
```

Comment:
These are all defaults and so there should be no need to specify them.

Reply:
Ok, I struggled a bit to understand that and thought it was nice to be explicit. Will remove